src/main/java/com/epam/ivankaliada/BasicSpringApp/service/EmployeeServiceImpl.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.ivankaliada.BasicSpringApp.service;
+
+import com.epam.ivankaliada.BasicSpringApp.entity.Employee;
+import com.epam.ivankaliada.BasicSpringApp.repository.EmployeeRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class EmployeeServiceImpl implements EmployeeService {
+
+    @Autowired
```

Comment:
There is a better way to inject dependencies into services.
https://www.baeldung.com/spring-injection-lombok#constructor-injection-with-lombok

Also, it will be difficult to write unit tests for such services, because there are no setters here.